em-bundling/values.yaml

```diff
@@ -52,10 +52,10 @@ rpe-service-auth-provider:
 
 dm-store:
   blobstorage:
-    enabled: true
+    enabled: false
   java:
     releaseNameOverride: chart-em-bundling-release-dm-store
-    image: hmctspublic.azurecr.io/dm/store:latest
+    image: hmcts.azurecr.io/hmcts/dm-store:pr-334  # fix for postgres bug
     keyVaults:
```

Comment:
keyvaults are not used in product charts - should use sealedsecrets instead. Define empty keyvaults value by default: https://github.com/hmcts/chart-cmc/blob/09dd8f268220a1c4557c144d6c30368dd904c222/cmc/values.yaml#L19 and specify the vault vars with env vars instead. A hacky script to grab secrets from vault and create sealedsecret: https://github.com/hmcts/chart-cmc/blob/master/k8s/bin/vault-to-sealedsecret.sh. These can then be commited to flux: https://github.com/hmcts/cnp-flux-config/tree/master/k8s/demo/common/money-claims and referenced: https://github.com/hmcts/cnp-flux-config/blob/7d768d53111268c2d428095af5679b20a4280ba3/k8s/demo/common/money-claims/cmc-demo.yaml#L27